src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationEvents.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace AspNet.Security.OAuth.SuperOffice
+{
+    /// <summary>
+    /// Default <see cref="SuperOfficeAuthenticationEvents"/> implementation.
+    /// </summary>
+    public class SuperOfficeAuthenticationEvents : OAuthEvents
```

Comment:
Do we have a concrete use case where this `ValidateIdToken` would be useful? If not, we should remove that and wait for actual user demand.

Reply:
The file reference here is SuperOfficeAuthenticationEvents, but ValidateIdToken could also be referred to as the Options property. 

Because this file is no longer part of the project, I'll commend on the options property. 

On one hand, "in the name of security" and because it's there, this is something that should always be validated. On the other hand, suppose a consumer simply prefers vanilla OAuth without the added security, should we prevent this option? 

It's default setting is true, so most folks should ignore it and only have minimalistic configuration settings... Benefits from having it enabled by default. 

A side thought: Being able to disable it might allow some people struggling with security to continue working through any issues in the development environment. But then it could be enforced in the production environment. Could put that in the options.Validate.